Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Use a lightweight base image
+FROM node:18-alpine
+
+# Set the working directory
+WORKDIR /app
+
+# Copy the package.json and package-lock.json files
+COPY package*.json ./
+
+# Install dependencies and clean npm's cache
+RUN npm install --production && \
+    npm cache clean --force
```

Comment:
I'm not sure that I fully understand the reason for this string inside docker, could you clarify?